docs/cloud/features/test-replay.mdx

```diff
@@ -114,6 +132,111 @@ Toggle each test attempt, play back errors at a variety of speeds or scrub throu
   alt="Test Replay scrubber controls"
 />
 
+## How Test Replay Works
+
+Our aim is to create an impactful debugging experience that covers the most ground for users.
+Cypress captures a wide range of data during end-to-end and component test runs.
+This data is then uploaded and stored in Cypress Cloud and replayed for debugging through Test Replay.
+
+### What is captured
+
+- Test commands
+- Network requests
+- Console logs
+- JavaScript errors
+- Element rendering
+- DOM mutations
+- CSS styles
+- SVG elements
+- Iframes
+- Shadow DOM
+- Canvas elements (v13.5.0+, please contact us at support@cypress.io to enable capturing canvas elements)
+
+### What is not captured
+
+Currently Test Replay does not support the following:
+
+- Tests recorded in Cypress App below v13.
+- WebKit browser (Safari) ([see issue](https://github.com/cypress-io/cypress/issues/28895))
+- Firefox browser ([see issue](https://github.com/cypress-io/cypress/issues/28894))
+- Video ([see issue](https://github.com/cypress-io/cypress/issues/29903))
+- Audio ([see issue](https://github.com/cypress-io/cypress/issues/29905))
+- Objects that do not have `type="image/svg+xml"` ([see issue](https://github.com/cypress-io/cypress/issues/29904))
+- Shadow DOM with `slotAssignment="manual"` ([see issue](https://github.com/cypress-io/cypress/issues/29107))
+- Shadow DOM canvas ([see issue](https://github.com/cypress-io/cypress/issues/30438))
```

Comment:
```suggestion
- Shadow DOM canvases ([see issue](https://github.com/cypress-io/cypress/issues/30438))
```

Reply:
Updated here to be more specific on plurality and elements vs other things [`56e77c8` (#5985)](https://github.com/cypress-io/cypress-documentation/pull/5985/commits/56e77c8547c7ac7c4ac3752e6e44a73170bfc15c)